# calibration data of camera 01

#invpol : [-50.223848022188406, 2.081401802641805e+02, -2.865465242831615e+02, 74.906886446105820, -4.256847434454998e+02, 8.040273572856454e+02]
#mapcoef : [5.783653041151614e+02 ,0 ,-7.808561561145619e-04 ,5.647614979301165e-07 ,-8.856190106917469e-10]
#yc : 5.377651999093430e+02
#xc : 9.397645009259877e+02
#c: 1
#d : 0
#e : 0
#H_res : 1080
#pixel_length : 0.0014


invpol : [-58.437369736305860, 2.444633606636849e+02, -3.442317329618735e+02, 1.108366545869685e+02, -4.281865259689594e+02, 7.998813162057033e+02]
mapcoef : [5.778042729620329e+02 ,0 ,-7.871474054011615e-04 ,6.056284600923182e-07 ,-9.543575918132550e-10]
yc : 5.440055025943838e+02
xc : 9.530229815813301e+02
c: 1
d : 0
e : 0
H_res : 512
pixel_length : 0.0014





